src/Xamarin.Android.Tools.Bytecode/Tests/Xamarin.Android.Tools.Bytecode-Tests.csproj

```diff
@@ -56,7 +56,13 @@
     <Compile Include="ParameterFixupTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <TestJar Include="java\**\*.java" />
+    <TestJar
+        Include="java\**\*.java"
+        Exclude="java\java\util\Collection.java"
```

Comment:
I'm open to suggestions for alternate strategies.

> What if they are mutually dependent on the ones in @(TestJar) and @(TestJarNoParameters) ?

Try to avoid that? Or we could introduce a third item group which is compiled after those other item groups... It's solvable. Not necessarily "pretty" or "sane," but solvable.